### ISSUE TITLE
Docs - Inconsistency when wrapping content inside some tables

### ISSUE THREAD

**Issue by mathbraga** [apollographql/apollo-server#7766]
## Description

Some tables in the docs seem to not wrap its content correctly and obstruct part of the text, making readers have to scroll back and forth for long content. Makes UI look inconsistent between tables.

### Screenshots for clarification

- **Content not wrapping**
![image](https://github.com/apollographql/apollo-server/assets/31048764/32be9d71-ba35-49ff-9bd3-3d666f1a4f77)

- **With wrap**
![image](https://github.com/apollographql/apollo-server/assets/31048764/a8754cf4-7eb2-4e7c-9ace-325b8776ab1e)

### Short video for better showcase

https://github.com/apollographql/apollo-server/assets/31048764/2afac57f-99be-4ff7-aad1-fccb1ca96a46

### Relevant environment info
**Resolution:** 1920 x 1080

**Browsers tested:**
- Brave v1.59.120
- Edge 118.0.2088.61

**Comment by trevor-scheer** [comment]
Yeah, this isn't great and something I've noticed before. We didn't come up with a great solution when I brought it up last time. This is due to a combination of long option names (which widens the first column) and long words which can't be broken over a new line in the second column.

For now, you'll probably find the "wide mode" option desirable. You can find it up at the top right next to dark/light mode.
